### DISCUSSION TITLE
Accensione luci non funziona dopo aver configurato duck DNS

### DISCUSSION THREAD

**Discussion by Alby1825** [indomus/forum#3084] [HUB: Home Assistant]
Trovo Home Assistant molto interessante e sto cercando di documentarmi. Ho seguito le guide per poter accedere da remoto ad HA e tutto sembra aver funzionato. Non mi funzionano più gli interruttori luci presenti sulla plancia. Ho un gateway Bticino e interruttori connessi e tutto funzionava prima di installare DuckDNS. Per contro il termostato sempre Bticino riesco a comandarlo da HA anche facendo la controprova entrando nell'app Control Bticino. Spero di essermi spiegato bene. grazie per l'eventuale aiuto.

**Comment by snakuzzo** [comment]
E cosa c'entrerebbe il fatto che non sei programmatore ? Mica per usare HA serve saper programmare ?
Infine, DuckDNS non c'entra assolutamente nulla con il tuo problema...ed in base ai dettagli forniti non credo ti si possa aiutare.
Prova a postare i log di Core (o di eventuali addon legati a quella roba Bticino) nel momento in cui provi a pilotare le entità di cui parli

**Comment by Alby1825** [comment]
correggo subito

**Comment by Alby1825** [comment]
Questa è la voce del registro, spero di aver postato giusto

Registratore: homeassistant.helpers.service
Fonte: helpers/service.py:279
Prima occorrenza: 07:40:27 (11 occorrenze)
Ultima registrazione: 11:35:00

Referenced entities switch.luce_corridoio are missing or not currently available
Referenced entities switch.luce_cucina are missing or not currently available

**Reply by snakuzzo** [reply]
Quelle entità non esistono o sono non disponibili.
Dovresti verificare come le hai integrate

**Comment by Alby1825** [comment]
Ho un preblema con duckDNS che non so per qual motivo ha smesso di funzionare,  ho provato a cercare giorni e giorni ma non riesco ad uscirne...  il registro mi da questo: 

s6-rc: info: service legacy-services: stopping
s6-rc: info: service legacy-services successfully stopped
s6-rc: info: service duckdns: stopping
[09:21:31] INFO: Service duckdns exited with code 256 (by signal 15)
s6-rc: info: service duckdns successfully stopped
s6-rc: info: service legacy-cont-init: stopping
s6-rc: info: service legacy-cont-init successfully stopped
s6-rc: info: service fix-attrs: stopping
s6-rc: info: service fix-attrs successfully stopped
s6-rc: info: service s6rc-oneshot-runner: stopping
s6-rc: info: service s6rc-oneshot-runner successfully stopped
s6-rc: info: service s6rc-oneshot-runner: starting
s6-rc: info: service s6rc-oneshot-runner successfully started
s6-rc: info: service fix-attrs: starting
s6-rc: info: service fix-attrs successfully started
s6-rc: info: service legacy-cont-init: starting
s6-rc: info: service legacy-cont-init successfully started
s6-rc: info: service duckdns: starting
s6-rc: info: service duckdns successfully started
s6-rc: info: service legacy-services: starting
s6-rc: info: service legacy-services successfully started
# INFO: Using main config file /data/workdir/config
+ Account already registered!
[09:21:44] INFO: Starting DuckDNS...
[09:21:44] WARNING: KO
[09:21:44] INFO: Renew certificate for domains: MIODOMINIO.duckdns.org/ and aliases: 
# INFO: Using main config file /data/workdir/config
Processing MIODOMINIO.duckdns.org/
 + Checking domain name(s) of existing cert... changed!
 + Domain name(s) are not matching!
 + Names in old certificate: MIODOMINIO.duckdns.org
 + Configured names: MIODOMINIO.duckdns.org/
 + Forcing renew.
 + Checking expire date of existing cert...
 + Valid till Oct  7 05:57:31 2024 GMT (Longer than 30 days). Ignoring because renew was forced!
 + Signing domains...
 + Generating private key...
 + Generating signing request...
req: Missing '=' after RDN type string '/' in subject name string


grazie
